Assets/Plugins/iOS/mParticleUnity.m

```diff
@@ -468,11 +468,11 @@ void _Upload() {
             return toChar(@"{}");
         }
     }
-    
+
     char* _User_GetUserIdentities(const char *mpid) {
         @try {
             NSString *mpidString = [[NSString alloc] initWithCString:mpid encoding:NSUTF8StringEncoding];
-            NSDictionary *userIdentities = [[[MParticle sharedInstance].identity getUser:(@([mpidString longLongValue]))] userIdentities];
+            NSDictionary *userIdentities = [[[MParticle sharedInstance].identity getUser:(@([mpidString longLongValue]))] identities];
```

Comment:
I changed this due to the error I was receiving

	No visible @interface for 'MParticleUser' declares the selector 'userIdentities'

Reply:
@MrTravisB this is the correct change, detailed here: https://github.com/mParticle/mparticle-apple-sdk/blob/master/migration-guide-v8.md#removal-of-idfa-and-updated-identity-api

Ideally we update the MParticle.cs to have an `identities` api rather than userIdentities, but this will suit for now.